eng/testing/mobile/tests.targets

```diff
@@ -1,11 +1,23 @@
 <Project>
+  <Import Project="$(MSBuildThisFileDirectory)runscript.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)outerbuild.targets" Condition="'$(IsCrossTargetingBuild)' == 'true'" />
```

Comment:
nit: `outerBuild.targets` (test build was failing on case-sensitive filesystems)

Reply:
thanks, that saved me some time :)